### PR TITLE
Feature/firebase login

### PR DESCRIPTION
**Pull request by raxitp02**
Firebase configuration, login and register

### CODE REVIEW

**Review by Sharmil7171 — APPROVED** [review]

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
```
Comment (Dhruvkumarshah) [inline public/index.html:2]: Why do we need this file?

File: src/app/test/test.service.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import { AngularFireDatabase } from '@angular/fire/database';
+
+@Injectable({
```
Comment (Dhruvkumarshah) [inline src/app/test/test.service.ts:4]: Do we need this file? if yes, create services folder have all the services inside that folder as well as rename the service name

File: src/environments/environment.prod.ts [hunk 3]
```diff
@@ -1,3 +1,13 @@
 export const environment = {
-  production: true
+  production: true,
+  firebaseConfig :{
```
Comment (Dhruvkumarshah) [inline src/environments/environment.prod.ts:3]: looks like settings are same for prod & non-prod. have only env.ts file or else create two separate configuration for prod & non-prod

File: src/app/app-routing.module.ts [hunk 4]
```diff
@@ -5,6 +5,14 @@ const routes: Routes = [
   {
     path: '',
     loadChildren: () => import('./tabs/tabs.module').then(m => m.TabsPageModule)
+  },
```
Comment (Dhruvkumarshah) [inline src/app/app-routing.module.ts:8]: I do not see login page on app load

**Comment by Dhruvkumarshah** [comment]
please have password field hidden

**Pull request merged**